xml/ha_maintenance.xml

```diff
@@ -0,0 +1,375 @@
+    </para>
+    <para>
+     From this point on, the resource will be managed by the &hasi; software
+     again.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.ha.manual_config.cli.maint.mode">
+  <title>Using Maintenance Modes with &crmsh;</title>
+
+  <variablelist>
+   <varlistentry>
+    <term>Applying Maintenance Mode to your Cluster</term>
+    <listitem>
+     <para>
+      In case you want to put the whole cluster in maintenance mode, use the
+      following command:
+     </para>
+<screen>&prompt.root;<command>crm</command> configure property maintenance-mode=true</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Applying Maintenance Mode to Nodes</term>
+    <listitem>
+     <para>
+      To put a node into maintenance mode (for example, the node
+      <literal>&node1;</literal>):
+     </para>
+<screen>&prompt.root;<command>crm</command> node maintenance &node1;</screen>
+     <para>
+      The <command>crm status</command> command will show the maintenance mode
+      for &node1; and no more resources will be allocated to that node. To
+      remove the maintenance flag from the node, use:
+     </para>
+<screen>&prompt.root;<command>crm</command> node ready &node1;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Applying Maintenance Mode to Resources</term>
+    <listitem>
+     <para>
+      To put a specific resource into maintenance mode (for example, the
+      resource <literal>ipaddress</literal>) enter:
+     </para>
+<screen>&prompt.root;<command>crm</command> resource maintenance ipaddress true</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Putting a Resource Into Unmanaged Mode</term>
+    <listitem>
+     <para>
+      To put a specific resource into unmanaged mode, enter:
+     </para>
+<screen>&prompt.root;<command>crm</command> resource unmanage <replaceable>RESOURCE_ID</replaceable></screen>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+ <sect1>
+  <title>Rebooting a Cluster Node</title>
+  <para>If you need to reboot a cluster node for a maintenance task, it is not a
+   good idea to simply shut down the node down without precautions. Rebooting a cluster
+   node (as well as stopping the cluster stack on a node) will trigger the
+   normal &ha; recovery process&mdash;like in the case of a reboot caused by a fault.
+   That means the node will be fenced. Services that had been running on
+   this node will be restarted on other nodes (or migrate back to the same node
+   when it has finished rebooting and has rejoined the cluster).
+ </para>
+<para>If the objective is to move services off the node in an orderly fashion,
+the best way to do that is to set the node to standby before shutting
+down the cluster stack and rebooting.
+
+That way, services can migrate off the node without having to wait for
+fencing to confirm that the node really is no longer a member in the
```

Comment:
That way, services can migrate off the node without concerning about reaching the shutdown timeout of pacemaker.

Reply:
Standby mode is officially described in here:
http://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/1.1/html/Pacemaker_Explained/_moving_resources.html#_moving_resources_manually

Basically putting a node into "standby" mode is to "move any resources running on the node away from here and prevent the node from hosting any resources".

Reply:
If you want to shutdown/reboot the machine, you'd prefer to manually perform "systemctl stop pacemaker.service" first before shutting down the node.

-- I was thinking to suggest so for the case that pacemaker.sevice is not enabled by "systemctl enable pacemaker" so that it was started manually. But now I think it'd be better that we always suggest so.